articles/quickstart/webapp/aspnet-core-beta/01-login.md

```diff
@@ -0,0 +1,242 @@
+
+The SDK requires that the JSON Web Token (JWT) be signed with an asymmetric key. To configure this go to the settings for your application in the Auth0 Dashboard, scroll down and click on **Show Advanced Settings**. Go to the **OAuth** tab and ensure the **JsonWebToken Signature Algorithm** is set to **RS256**.
+
+Save your changes.
+
+## Configure Your Application to Use Auth0
+
+[Universal Login](/hosted-pages/login) is the easiest way to set up authentication in your application. We recommend using it for the best experience, best security and the fullest array of features. This guide will use it to provide a way for your users to log in to your ASP.NET Core application.
+
+### Install dependencies
+
+To integrate Auth0 with ASP.NET Core you can use our beta SDK by installing the `Auth0.AspNetCore.Mvc` Nuget package to your application.
+
+```bash
+Install-Package Auth0.AspNetCore.Mvc -IncludePrerelease
+```
+
+### Install and configure the SDK
+
+To enable authentication in your ASP.NET Core application, use the middleware provided by the SDK.
+Go to the `ConfigureServices` method of your `Startup` class and call `services.AddAuth0Mvc()` to configure the Auth0 ASP.NET Core SDK.
+
+Ensure to configure the `Domain` and `ClientId`, these are required fields to ensure the SDK knows which Auth0 tenant and application it should use.
+
+By default, the SDK requests the `openid` and `profile` scopes. If needed, you can request different scopes by setting the option's `scope` property.
+
+::: note
+In the code sample below, only the `openid` scope is requested.
+:::
+
+```cs
+// Startup.cs
+
+public void ConfigureServices(IServiceCollection services)
+{
+    // Cookie configuration for HTTP to support cookies with SameSite=None
+    services.ConfigureSameSiteNoneCookies();
+
+    // Cookie configuration for HTTPS
+    // services.Configure<CookiePolicyOptions>(options =>
+    // {
+    //    options.MinimumSameSitePolicy = SameSiteMode.None
+    // });
+
+    // Add authentication services
+    services
+        .AddAuth0Mvc(options => {
+            // Set the authority to your Auth0 domain
+            options.Domain = Configuration["Auth0:Domain"];
+            // Configure the Auth0 Client ID
+            options.ClientId = Configuration["Auth0:ClientId"];
+            // Change the scopes used
+            options.Scope = "openid";
+        });
+
+    // Add framework services.
+    services.AddControllersWithViews();
+}
+```
+
+::: note
+The `ConfigureSameSiteNoneCookies` method used above was added as part of the [sample application](https://github.com/auth0-samples/auth0-aspnetcore-mvc-samples/blob/master/Quickstart/01-Login/Support/SameSiteServiceCollectionExtensions.cs) in order to ([make cookies with SameSite=None work over HTTP when using Chrome](https://blog.chromium.org/2019/10/developers-get-ready-for-new.html)). We recommend using HTTPS instead of HTTP, which removes the need for the `ConfigureSameSiteNoneCookies` method.
+:::
```

Comment:
Should we be providing/suggesting how to work authentication unsecurely, even if on development?  The decision on express was to provide instructions on using it with https and linking to https://auth0.com/docs/libraries/secure-local-development

Reply:
Fwiw, we have removed the secure localhost guidance in express for now since Chrome has allowed for a 2minute allowance of the samesite=none cookie https://www.chromestatus.com/feature/5088147346030592

Reply:
I am not sure I get the question. Personally, I would prefer to stick to HTTPS, as .NET comes with https configured out of the box (it literally requires a user action to use HTTP locally instead of HTTPS). It would also remove the SameSiteNone guidance which I believe is not something we want to be adding guidance on anyway.

However, I used HTTP as this was discussed a few months back and I was asked to use `HTTP` and not `HTTPS`. Not sure why anymore, but happy to use HTTPS here if you believe that is the way to go!